### ISSUE TITLE
Navigator position is incorrect

### ISSUE THREAD

**Issue by kamil-musialowski** [highcharts/highcharts#21299]
#### Expected behaviour
Distance between the axis and navigator should be equal to the value set in config.

#### Actual behaviour
Navigator overlaps the xAxis when margin is set to 0.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/n1rvwet9/

#### Product version
Stock 11.4.3

Broken since 11.0.0, worked fine in 10.3.3
Bisected to https://github.com/highcharts/highcharts/commit/0480dc0c204b25a48a851d93bd12aec3720dc2e0

**Comment by pawelfus** [comment]
*Internal note:*
Needs a bisect

**Comment by kamil-musialowski** [comment]
Bisected to https://github.com/highcharts/highcharts/commit/0480dc0c204b25a48a851d93bd12aec3720dc2e0

**Comment by pawelfus** [comment]
Yes, this could be it. It looks like a workaround for incorrect margin=0 🤔 Thanks for the bisect!

**Comment by magdalena-gut** [comment]
> Yes, this could be it. It looks like a workaround for incorrect margin=0 🤔 Thanks for the bisect!

I don't see a clear reason for this workaround - it might have been needed in earlier versions. I've set the default margin to 0 according to the [API](https://api.highcharts.com/highstock/scrollbar.margin)

**Comment by pawelfus** [comment]
Hm.. it seems to be a design change, an expected one. Perhaps we should update the default value in API. Do you agree @TorsteinHonsi ?

**Comment by magdalena-gut** [comment]
What do you think of this one, @TorsteinHonsi?

**Comment by TorsteinHonsi** [comment]
@magdalena-gut We should probably update the default value like @pawelfus  says. But it is also something clearly wrong because setting it to 0 doesn't do what the [docs says](https://api.highcharts.com/highstock/navigator.margin). So probably we should try to make this behave as documented without breaking the default behavior in the visual tests .
